InnerEye/ML/dataset/full_image_dataset.py

```diff
@@ -270,7 +270,7 @@ def convert_channels_to_file_paths(channels: List[str],
                                    rows: pd.DataFrame,
                                    local_dataset_root_folder: Path,
                                    patient_id: str,
-                                   allow_incomplete_labels: bool = False) -> Tuple[List[Path], str]:
+                                   allow_incomplete_labels: bool = False) -> Tuple[List[Optional[Path]], str]:
     """
     Returns: 1) The full path for files specified in the training, validation and testing datasets, and
              2) Missing channels or missing files.
```

Comment:
Please clarify what "missing channels or missing files" is. By most people's reading, I think they would expect a list